solutions/src/test/java/ru/sberbank/school/task11/StreamsTest.java

```diff
@@ -0,0 +1,60 @@
+package ru.sberbank.school.task11;
+
+import org.junit.jupiter.api.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class StreamsTest {
+
+    private final static List<String> TEST_VALUES = Arrays.asList("1", "2", "3", "4", "5", "6", "7");
```

Comment:
&#39;static&#39; modifier out of order with the JLS suggestions.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1668701417>*